marshmallow_recipe/generics.py

```diff
@@ -0,0 +1,117 @@
+import dataclasses
+import typing
+from dataclasses import Field
+from types import GenericAlias, UnionType
```

Comment:
I suggest using the same import startegy as it is done in bake.py for this package.

Reply:
fixed